content/zh/docs/topics/charts.md

```diff
@@ -607,7 +607,7 @@ storage: "gcs"
 **注意：** 如果`helm install`或`helm upgrade`使用了`--set`参数，这些值在客户端会被简单地转换为YAML。
 
 **注意：** 如果values 文件存在任何必需的条目，它们会在chart模板中使用['required'
-函数](http://helm.sh/docs/howto/charts_tips_and_tricks.md) 声明为必需的。
+函数](http://docs.helm.sh/zh/docs/howto/charts_tips_and_tricks.md) 声明为必需的。
```

Comment:
Don't end with .md - this URL will be https://helm.sh/zh/docs/howto/charts_tips_and_tricks/ but /docs/howto/charts_tips_and_tricks/  is fine too.

Reply:
OK，I need to distinguish between translation，URL will be https://helm.sh/zh/docs/howto/charts_tips_and_tricks/